src/dsp_tools/xmllib/models/licenses/recommended.py

```diff
@@ -3,7 +3,7 @@
 from enum import StrEnum
 
 
-class License: ...
+class License(StrEnum): ...
```

Comment:
This is necessary in order to get a useful output of `find_license_in_string()`. Before, you couldn't do anything useful with the returned `License`, now you can retrieve its actual value with `lic.value`.